src/igvfd/schemas/in_vitro_system.json

```diff
@@ -0,0 +1,195 @@
+        },
+        {
+            "$ref": "mixins.json#/url"
+        },
+        {
+            "$ref": "mixins.json#/taxa"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "biosample.json#/dependentSchemas"
+        ],
+        "passage_number": {
+            "comment": "Only in vitro system with classification as a cell line are allowed to have passage_number property.",
+            "properties": {
+                "classification": {
+                    "enum": [
+                        "cell line"
+                    ]
+                }
+            }
+        }
+    },
+    "dependentRequired": {
+        "time_post_factors_introduction": [
+            "time_post_factors_introduction_units"
+        ],
+        "time_post_factors_introduction_units": [
+            "time_post_factors_introduction"
+        ]
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "classification": {
+            "title": "Classification",
+            "description": "The general category of this type of in vitro system.",
+            "comment": "Required property. The specified category should match the ontology used to describe this type of in vitro system.",
+            "permission": "import_items",
+            "type": "string",
+            "enum": [
+                "organoid",
+                "gastruloid",
+                "embryoid",
+                "cell line",
+                "differentiated cell",
+                "differentiated tissue",
+                "reprogrammed cell"
+            ]
+        },
+        "time_post_factors_introduction": {
+            "title": "Time Post Factors Introduction",
+            "description": "The time that elapsed past the point when factors were introduced.",
+            "comment": "Use in conjunction with time_post_factors_introduction_units to specify time elapsed post factors introduction. Use of this property requires time_post_factors_introduction_units specification.",
+            "type": "number"
+        },
+        "time_post_factors_introduction_units": {
+            "title": "Time Post Factors Introduction Units",
+            "description": "The units of time that elapsed past the point when factors were introduced.",
+            "comment": "Use in conjunction with time_post_factors_introduction to specify time elapsed post factors introduction. Use of this property requires time_post_factors_introduction specification.",
+            "type": "string",
+            "enum": [
+                "minute",
+                "hour",
+                "day",
+                "week",
+                "month"
+            ]
+        },
+        "date_obtained": {
+            "title": "Date Collected"
+        },
+        "introduced_factors": {
+            "title": "Introduced Factors",
+            "description": "A list of treatments applied to the biosample that introduces factors.",
```

Comment:
description: "A list of treatments applied to the biosample."

Reply:
done